templates/userprofile/details.html

```diff
@@ -34,40 +34,41 @@
     <div class="tab-content col-lg-10 offset-lg-1">
       <div class="tab-pane active" id="orders" role="tabpanel">
         <div id="accordion" role="tablist" aria-multiselectable="true">
-          <div class="userprofile__orders-header">
-            <div class="row">
-              <div class="col-md-2 col-3"><h3>{% trans "Order" %}</h3></div>
-              <div class="col-md-3  hidden-sm-down"><h3>{% trans "Date" %}</h3></div>
-              <div class="col-md-2 col-6"><h3 class="float-md-right">{% trans "Summary" %}</h3></div>
-              <div class="col-md-4  hidden-sm-down"><h3>{% trans "Status" %}</h3></div>
-              <div class="col-md-1 col-3"></div>
-            </div>
-          </div>
-          {% for order in orders %}
-          <div class="userprofile__orders">
-            <div class="userprofile__orders__row">
+          {% if orders|length == 0 %}
```

Comment:
For loop doesn't cover table header so it'll be still visible if there are no orders.